handbook/product/onboarding/index.md

```diff
@@ -15,13 +15,17 @@ Welcome to Sourcegraph! As a member of the product team, it is your job to be th
 ## Product team member checklist
 
 - Get to know our team and learn about Sourcegraph (company and product)
+  - Schedule 30-min 1:1s with your manager for the first three days at Sourcegraph to keep up with your onboarding and ask all the questions you might have
```

Comment:
I'd suggest something less specific: "Schedule 1:1s with your manager for your first week to keep up with your onboarding..."
My first 3 1:1s were MWF, I think it might look a bit different for different people.

Reply:
I think it makes most sense to put this under the above Manager checklist - that way the manager is blocking time to help onboard the new teammate throughout their first week.

```suggestion
  - Schedule daily check-ins for the first week at Sourcegraph to keep up with your onboarding and to create space for answering any questions that might come up.
```